src/v0.8/core/filecoin/Filecoin.sol

```diff
@@ -18,50 +18,75 @@
 
 pragma solidity ^0.8.21;
 
-import {MarketAPI} from "@zondax/filecoin-solidity/contracts/v0.8/MarketAPI.sol";
```

Comment:
Filecoin-solidity Error: Contract `src/v0.8/core/filecoin/Filecoin.sol:Filecoin` is not upgrade safe
@zondax/filecoin-solidity/contracts/v0.8/utils/Actor.sol:120: Use of delegatecall is not allowed
It will be processed when the filecoin function is improved.